style.css

```diff
@@ -64,4 +64,147 @@ body
     {
         display: none;
     }
+}
+
+/* Second section top cover */
+
+.top-cover
+{
+    position: relative;
+    height: 800px;
+    background-image: url(/images/cover-background-black-tower.jpg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    max-width: 100%;
+}
+
+.navbar
+{
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
```

Comment:
Actually what is the issue here, you have a container and you need the 2 contents inside it should be in both sides. so what is the method for that.

Reply:
justify-content: space-between;